modes/longitudinal/src/toolbarButtons.js

```diff
@@ -286,6 +286,23 @@ const toolbarButtons = [
     id: 'Layout',
     type: 'ohif.layoutSelector',
   },
+  // Todo: MPR not ready yet for SEG support, not activating it now
+  // {
+  //   id: 'MPR',
+  //   type: 'ohif.action',
+  //   props: {
+  //     icon: 'old-play',
+  //     label: 'MPR',
+  //     type: 'action',
+  //     commands: [
+  //       {
+  //         commandName: 'activateMPRForActiveViewport',
+  //         commandOptions: {},
```

Comment:
I suggest defining a module exported hanging protocol name here that can be applied.  That way, all hanging protocols are defined in the same way, and a mode can changes the "MPR" layout to whatever they want just by switching the HP to apply.    I would also suggest specifying the default stage to start with, at least as a possible option.